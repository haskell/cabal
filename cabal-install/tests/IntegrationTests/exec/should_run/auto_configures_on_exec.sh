source ../common.sh
cabal exec echo find_me_in_output
