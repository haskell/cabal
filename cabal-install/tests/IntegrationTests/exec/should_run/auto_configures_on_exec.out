Config file path source is commandline option.
Config file config-file not found.
Writing default configuration to config-file
find_me_in_output
